lib/gui/app.js

```diff
@@ -43,6 +43,7 @@ const app = angular.module('Etcher', [
 
   // Models
   require('./models/selection-state'),
+  require('./models/drive-constraints'),
```

Comment:
This is not needed, since the `DriveConstraintsModel` is not being directly used by the `.run()` blocks from this file.